pkg/action/action_handler.go

```diff
@@ -112,6 +114,9 @@ func (h *ActionHandler) registerActionExecutors() {
 	horizontalScaler := executor.NewHorizontalScaler(h.config.kubeClient, h.config.broker, h.scheduler)
 	h.actionExecutors[turboaction.ActionProvision] = horizontalScaler
 	h.actionExecutors[turboaction.ActionUnbind] = horizontalScaler
+
+	containerResizer := executor.NewContainerResizer(h.config.kubeClient, h.config.kubeletClient, h.config.broker, h.config.k8sVersion, h.config.noneSchedulerName, h.lockMap)
```

Comment:
If you make `ActionHandlerConfig` an interface, you can do something like
```
containerResizer := executor.NewContainerResizer(resizeHandlerConfig, h.lockMap)
```
Just don't passed in so many parameters.